src/Transfer/Sources/Supabase.php

```diff
@@ -224,6 +224,7 @@ public function __construct(string $endpoint, string $key, string $host, string
         $this->port = $port;
 
         $this->headers['Authorization'] = 'Bearer '.$this->key;
+        $this->headers['apiKey'] = $this->key;
```

Comment:
Do they require both bearer token and an API key header?

Reply:
Yeah, no idea why they do that. My assumption is that Kong wants the Authorization their Storage API behind it wants apiKey.

Reply:
Ah yep that'd make sense 👍